monitoring/mongodb/dashboard.py

```diff
@@ -518,7 +518,7 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     orientation="horizontal",
     targets=[
         Target(
-            expr="sum(" + Metrics.UP() + ") by (pod)",
+            expr="sum(" + Metrics.UP() + ") by (job)",
```

Comment:
should not be changed : the legend is `{{pod}}`, so the sum must be by `pod` as well (since the result series will not that label)

i.e. either:
```suggestion
            expr="sum(" + Metrics.UP() + ") by (pod)",
            templateLabel = "{{pod}}"
```
or
```suggestion
            expr="sum(" + Metrics.UP() + ") by (job)",
            templateLabel = "{{job}}"
```